test/esm.test.ts

```diff
@@ -0,0 +1,59 @@
+import { expect, it } from 'vitest'
+import { SvgPacker } from '../src'
+
+it('svg-packer in Node', async () => {
+  const result = await SvgPacker({
+    icons: [{
+      svg: `<svg viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg">
+  <circle cx="50" cy="50" r="50" />
+</svg>`,
+      name: 'icon1',
+    }],
```

Comment:
Extatct the icons to a module and ise the icons at root index html, add both icons to the dom in the browser